package NSteam.NProto;

message TSnippet
{
    optional bytes Query = 1;
    optional bytes B64QTree = 2;
    optional bytes Url = 3;
    optional bytes Region = 4;
    optional bytes TitleText = 5;
    repeated TFragment Fragments = 6;
    optional bytes Lines = 7;
    optional bytes ExtraInfo = 8;
    optional bytes Headline = 9;
    optional bytes HilitedUrl = 10;
    optional bytes HeadlineSrc = 11;
}

message TFragment
{
    optional bytes Text = 1;
}
